crawl-ref/source/dat/des/altar/ecumenical.des

```diff
@@ -419,3 +419,26 @@ x-----x
 x-----x
 xxb+bxx
 ENDMAP
+
+NAME:   dilly_ecumenical_altar_fadedgnrine
+TAGS:   no_monster_gen transparent
+MONS:   plant, gnoll; nothing
+SUBST: z = t111
+KITEM: E = never_decay gnoll shaman skeleton / never_decay gnoll skeleton
```

Comment:
```suggestion
KITEM: E = never_decay gnoll shaman skeleton / never_decay gnoll skeleton
KPROP: E = no_tele_into
```

This is a case where `no_tele_into` is vital, since it's a teleport closet.